build/tools/ortelius.md

```diff
@@ -202,6 +202,47 @@ curl "http://127.0.0.1:8080/v2/aggregates?startTime=2020-09-21T00:00:00Z&endTime
 }
 ```
 
+#### TxFee Aggregate
+
+AVAX Aggregate txfee
+
+**Example Call**
+
+```text
+curl "http://127.0.0.1:8080/v2/txfeeAggregates?startTime=2020-09-21T00:00:00Z&endTime=2020-10-21T00:00:00Z"
+```
+
+**Example Response**
+
+```javascript
+{
+  "aggregates": {
+    "startTime": "2020-09-21T00:00:00Z",
+    "endTime": "2020-10-21T00:00:00Z",
+    "txfee": "134818000000"
+  },
+  "startTime": "2020-09-21T00:00:00Z",
+  "endTime": "2020-10-21T00:00:00Z"
+}
+```
+
+#### Address Chain
+
+Responds with the chains an address appears on.
+
+**Params**
+
+| Name | Type | Description | Default | Max |
+| :--- | :--- | :--- | :--- | :--- |
+| `address` | `address` | A address to filter results by. May be supplied multiple times. | None | None |
+
+
+**Example Call**
+
+```text
+curl "http://127.0.0.1:8080/v2/addressChains?address=X-fujiABC"
```

Comment:
Also use `localhost` here.

Also please add a `1` in this mock address for the separator constant: `X-fuji1ABC`